iotlabcli/rest.py

```diff
@@ -330,6 +331,8 @@ def method(self, url, method='get',  # pylint:disable=too-many-arguments
                             json=json, files=files)
         if requests.codes.ok == req.status_code:
             return req.content if raw else req.json()
+        if requests.codes.no_content == req.status_code:
+            return {}
```

Comment:
returning an empty dict `{}` (as if the APi returned an empty JSON object `{}` in the case of `no content` might be opinionated. The rest of cli-tools (print_results, etc,) all expect some kind of JSON object so either we return `{}`, we might `return` (same as `return None`), but in that case the print_results must be OK with it (maybe show nothing in the command line), and what about jmespath that expects json...